hamilton_launch_server/dockerize.sh

```diff
@@ -54,9 +58,55 @@ build_rasperry_pi() {
         $IMAGE_NAME go build
 }
 
+run_ffserver() {
+    cd $LIVESTREAM_DIR
+    ls -alh
+    docker run --rm \
+        --name ffserver \
+        --volume "$LIVESTREAM_DIR:$LIVESTREAM_DOCKER_HOME" \
+        --network livestream-net \
+        -p 8090:8090 \
+        $LIVESTREAM_IMAGE_NAME ffserver -f ffserver.conf
+    cd ..
```

Comment:
It would be better to do this in a subshell

Reply:
Sorry, what do you mean?

Reply:
```
(cd $LIVESTREAM_DIR &&
    docker run --rm \
        --name ffserver \
        --volume "$LIVESTREAM_DIR:$LIVESTREAM_DOCKER_HOME" \
        --network livestream-net \
        -p 8090:8090 \
        $LIVESTREAM_IMAGE_NAME ffserver -f ffserver.conf
)
```